tests/test_aea_exectimeout.py

```diff
@@ -85,7 +85,7 @@ def handler_func(*args, **kwargs):
         behaviour_cls = make_behaviour_cls_from_funcion(handler_func)
 
         test_skill = Skill(
-            SkillConfig(name="test_skill"),
+            SkillConfig(name="test_skill", author="fetchai"),
```

Comment:
@solarw , this will then create the public id `fetchai/test_skill:0.1.0` for this skill. The version could also be set as an arg but defaults to `0.1.0` atm